ONSdigital/census-rm-case-api-2021
# Add batch ID column to uac qid link table

### NOTE: The travis build will fail till the case-processor branch is merged

# Motivation and Context
We need to be able to identify batches of unaddressed uac qid pairs, the easiest solution for this was to add in a UUID batch_id column which is set off the unaddressed request messages. This repo must match the case-processor change

# What has changed
- Add batch ID to uac qid links model

# How to test?
Build the case-processor on the linked branch, then build this branch. All tests should pass and the acceptance tests should still pass.

# Links
https://github.com/ONSdigital/census-rm-case-processor/pull/18
https://github.com/ONSdigital/census-rm-qid-batch-runner/pull/3
https://github.com/ONSdigital/census-rm-ops/pull/10
https://trello.com/c/sOyLOxiP/783-add-batch-identifier-to-unaddressed-qid-uac-pairs-5
